src/main/pages/che-7/overview/con_persistent-volume-configuration.adoc

```diff
@@ -28,8 +28,8 @@ User-defined PVCs are created with few modifications:
 
 - they are provisioned with generated names to guarantee that it is not conflicting with other PVCs in namespace;
 
-- subpaths of mount volumes that reference user-defined PVCs are prefixed with `{workspace id}/{PVC name}`.
-It is done to have the same data structure on PV on different PVC strategies(link:#how-the-subpaths-are-used[more]);
+- sub-paths of mount volumes that reference user-defined PVCs are prefixed with `{workspace id}/{PVC name}`.
```

Comment:
Do not hyphen. The rule of thumb is to not hyphen whenever you can (if it doesn't affect the meaning of the word or doesn't create ambiguity).